### ISSUE TITLE
check phpm executable

### ISSUE THREAD

**Issue by xw2423** [spf13/PIV#9]
in ftplugin/php.vim:98

before Map C-H to search phpm, check php executable,like below:

```
if executable('phpm')
    inoremap <buffer> <C-H> <ESC>:!phpm <C-R>=expand("<cword>")<CR><CR>
endif
```

**Comment by zenware** [comment]
I agree, I've been having errors with the script trying to load phpm lately, my plan to fix the issue was to get a phpm exec in the path somewhere but I can't seem to find any documentation on what the phpm executable actually is, so that's not very helpful.

Anyways +1 to this issue s this line is currently commented out of my file.
